exercises/roman-numerals/runtests.jl

```diff
@@ -0,0 +1,36 @@
+using Base.Test
+
+include("roman-numerals.jl")
+
+# NOTE: Got that one from Python's testset
+samples = Dict(
+    1 => "I",
+    2 => "II",
+    3 => "III",
+    4 => "IV",
+    5 => "V",
+    6 => "VI",
+    9 => "IX",
+    27 => "XXVII",
+    48 => "XLVIII",
+    59 => "LIX",
+    93 => "XCIII",
+    141 => "CXLI",
+    163 => "CLXIII",
+    402 => "CDII",
+    575 => "DLXXV",
+    911 => "CMXI",
+    1024 => "MXXIV",
+    3000 => "MMM"
+)
+
+@testset "basic transformations" begin
+    for sample in samples
```

Comment:
Instead of the `begin` with a loop inside, the output of this might be more readable if each sample had its own testset so one can immediately see which test fails in the test output.